### PR TITLE
Add a simple description to this policy

### PR DESCRIPTION
**Pull request by gaetanmaisse**
**Issue**

https://graviteedevops.atlassian.net/browse/DOC-145

**Description**

Add a simple description to this policy

---
**Gravitee.io Automatic Deployment**

🚀 A prerelease version of this package has been published on Gravitee's private artifactory, you can:
 - use it directly by updating your project with version: `1.5.1-DOC-145-add-readme-SNAPSHOT`
 - download it from Artifactory [here](https://odbxikk7vo-artifactory.services.clever-cloud.com/gravitee-snapshots/io/gravitee/policy/gravitee-policy-keyless/1.5.1-DOC-145-add-readme-SNAPSHOT/gravitee-policy-keyless-1.5.1-DOC-145-add-readme-SNAPSHOT.zip)

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -21,7 +21,12 @@ endif::[]
 
 == Description
 
+This security policy consider any request as valid and so will not block any.
```
Comment (ivan-gravitee) [inline README.adoc:24]:
```suggestion
This security policy does not block any requests as it considers them as valid by default.
```

File: README.adoc [hunk 2]
```diff
@@ -21,7 +21,12 @@ endif::[]
...
+During its execution the policy will set multiple attributes:
+
+* `application`: Anonymous application default value i.e. `1`
```
[the remark this answers is not in the document]
Comment (gaetanmaisse) [reply README.adoc:27]: For this one the meaning is `application` is always set to the anonymous application value which is equal to 1

[the remark this answers is not in the document]
Comment (ivan-gravitee) [reply README.adoc:27]: Sure, so something like this perhaps?

* `application`: Anonymous application value, which is equal to `1` by default.

[the remark this answers is not in the document]
Comment (ivan-gravitee) [reply README.adoc:27]: Even better

[the remark this answers is not in the document]
Comment (gaetanmaisse) [reply README.adoc]: Thanks! I just removed the `by default` as it's not the default value but THE value

**Review by ivan-gravitee — APPROVED** [review]
LGTM

**Pull request merged**